README.md

```diff
@@ -160,20 +150,14 @@ Datawow.setup do |config|
 end
 ```
 
-After configuration has been set. You can calling APIs by not insert token like this
-```ruby
-Datawow.[class].create({data: "image URL"})
-```
-The system will be looking for your token by automatically
-
 # Response
 
 Response is a module and it contain such as meta-data, message, status and data.
```

Comment:
Response is a module and it contains data including: data, meta-data, message, and status